commands/playwright-keplr.js

```diff
@@ -307,6 +308,29 @@ module.exports = {
     await element.type(value);
     await module.exports.waitUntilStable(page);
   },
+  async waitAndClickWithRetry(selector, options) {
+    const maxRetries = 5;
+    let retries = 0;
+
+    while (retries < maxRetries) {
+      try {
+        await module.exports.waitAndClick(
+          selector,
+          module.exports.keplrWindow(),
+          options,
+        );
+        return;
+      } catch (error) {
+        retries++;
```

Comment:
we should add a small delay between retries to avoid differences between slow and fast computes
```
page.waitForTimeout(100);
```

Reply:
Introducing retries was done to cater slow and fast computes. Adding delay will slow things down

Reply:
retries will run very quickly on a fast computer, but very slowly on a slow computer.
lets say an element appears after 300ms 
the slow computer might perform 5 retires in 350ms and find the item at the final retry
the fast computer might run the retries in 50ms and miss all the retires